### PR TITLE
Use relative path for files

### PR DESCRIPTION
**Pull request by emilienbai**
Just a suggestion ;-)

### CODE REVIEW

File: map-europe-waste-recycling/map_europe_waste_recycling.ipynb [hunk 1]
```diff
@@ -31,8 +31,8 @@
     "\n",
     "# Import datasets\n",
     "# You can find these files under: https://github.com/marie-lefevre/Environment-Graphs/tree/master/map-europe-waste-recycling\n",
-    "file1 = pd.read_excel (r'C:\\Users\\marie\\Desktop\\file1_excel.xlsx')\n",
-    "file2 = pd.read_excel (r'C:\\Users\\marie\\Desktop\\file2_excel.xlsx')\n"
+    "file1 = pd.read_excel (r'./file1_excel.xlsx')\n",
```
Comment (marie-lefevre) [inline map-europe-waste-recycling/map_europe_waste_recycling.ipynb:34]:
```suggestion
    "file1 = pd.read_excel (r'.\file1_excel.xlsx')\n",
```

File: map-europe-waste-recycling/map_europe_waste_recycling.ipynb [hunk 2]
```diff
@@ -31,8 +31,8 @@
...
+    "file2 = pd.read_excel (r'./file2_excel.xlsx')\n"
```
Comment (marie-lefevre) [inline map-europe-waste-recycling/map_europe_waste_recycling.ipynb:35]:
```suggestion
    "file2 = pd.read_excel (r'.\file2_excel.xlsx')\n"
```

**Review by marie-lefevre — CHANGES_REQUESTED** [review]
Please change / into \

File: map-europe-waste-recycling/map_europe_waste_recycling.ipynb (hunk 1 above)
Comment (emilienbai) [reply map-europe-waste-recycling/map_europe_waste_recycling.ipynb:34]: ok

File: map-europe-waste-recycling/map_europe_waste_recycling.ipynb (hunk 2 above)
Comment (emilienbai) [reply map-europe-waste-recycling/map_europe_waste_recycling.ipynb:35]: not ok ;p

**Pull request merged**